onnxruntime_extensions/_cuops.py

```diff
@@ -90,6 +90,17 @@ def serialize_attr(cls, attrs):
         return attr_data
 
 
+class StringRemove(CustomOp):
```

Comment:
Do you know if there is any similar op in other framework/library that can be a reference?

Reply:
Renamed to MaskedFill